db/migrate/20240120155434_create_residents.rb

```diff
@@ -0,0 +1,8 @@
+class CreateResidents < ActiveRecord::Migration[7.1]
+  def change
+    create_table :residents do |t|
+      t.string :name, null: false, index: { unique: true }
+      t.datetime :loaded_at, default: -> { 'CURRENT_TIMESTAMP' }
```

Comment:
Oops, I forgot to add the actual timestamp columns here, I'll add those in and force push.